### DISCUSSION TITLE
Destination Oracle Add Typing and Deduping

### DISCUSSION THREAD

**Discussion by jbfbell** [airbytehq/airbyte#37024] [Connector Questions]
We removed "normalization" from Destination Oracle. Normalization was a process which would extract data from the raw json format and cast the data into tables with the appropriate types. For other destinations, we implemented [Typing and Deduping](https://airbyte.com/blog/introducing-airbyte-destinations-v2-typing-deduping) to replace normalization and we will bring typing and deduplication to Oracle if there is sufficient interest

The purpose of this discussion is to gauge interest in bringing this functionality back to Oracle.

Without typing and deduping, you would need to integrate another tool, such as [dbt](https://clickhouse.com/docs/en/integrations/dbt) to accomplish the same result. While integrating another tool gives you greater customization over the data format, it comes with additional operational overhead.

Should Airbyte bring Typing and Deduping to Oracle?

**Comment by leorehm** [comment]
+1

We are planning to implement Airbyte to replicate data from various sources to Oracle. Not having typing and deduping would unfortunately be a dealbreaker

**Comment by push2prod** [comment]
+1
Would also love support for oracle 19c

**Reply by Emma-Levi** [reply]
+1
We ingest from multiple sources into oracle but without typing we can’t use the destination

[a comment by an automated account is not shown]
